__tests__/app.test.js

```diff
@@ -25,6 +26,17 @@ describe("/api/topics", () => {
 				});
 			});
 	});
-
 });
 
+describe("/api", () => {
+	it("GET: 200 - responds with an object with all available endpoints", () => {
+		return request(app)
+			.get("/api")
+			.expect(200)
+			.then(({ body }) => {
+				console.log(body)
```

Comment:
Don't forget to remove `console.log`s before submitting a PR 😄 We don't want them in production code.